README.md

```diff
@@ -911,16 +917,18 @@ It is also necessary to fill out the `google` stanza in the configuration file.
 
 ```hocon
 google {
-  authScheme = "service"
+  applicationName = "cromwell"
+  
+  cromwellAuthenticationScheme = "service_account"
 
-  // If authScheme is "user"
+  // If cromwellAuthenticationScheme is "user_account"
```

Comment:
apologies if I've already asked this at some point, but why the change? Isn't it implied that it's cromwell because it's in cromwell?